.github/workflows/label-approved.yml

```diff
@@ -0,0 +1,18 @@
+name: Update Pull Request Approval Labels
+on:
+  pull_request_review:
+  pull_request:
+    types: [synchronize]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  label-approved:
+    runs-on: ubuntu-latest
+    if: "github.repository_owner == 'NixOS'"
+    steps:
+      - uses: nix-community/label-approved@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          pr_number: ${{ github.event.issue.number || github.event.number }}
```

Comment:
Testing a pull_request_review with the attempted fix.

(Sorry, testing GitHub actions is a pain.)

Reply:
Hopefully last (noisy) test.